scopes/harmony/application/application.main.runtime.ts

```diff
@@ -222,7 +246,7 @@ export class ApplicationMain {
       logger
     );
     const appCmd = new AppCmd();
-    appCmd.commands = [new AppListCmd(application)];
+    appCmd.commands = [new AppListCmd(application), new RunCmd(application, logger)];
```

Comment:
how did it works before this??

Reply:
added new command: `bit app run <app name>` (in addition to `bit run <app name>`).
You are welcome